internal/hub/controllers/slicegateway_controller.go

```diff
@@ -190,6 +190,7 @@ func (r *SliceGwReconciler) Reconcile(ctx context.Context, req reconcile.Request
 		SliceGatewaySubnet:          sliceGw.Spec.LocalGatewayConfig.GatewaySubnet,
 		SliceGatewayRemoteSubnet:    sliceGw.Spec.RemoteGatewayConfig.GatewaySubnet,
 		SliceGatewayHostType:        sliceGw.Spec.GatewayHostType,
+		SliceGatewayNodePort:        sliceGw.Spec.LocalGatewayConfig.NodePort,
```

Comment:
Do we already have this field in the CRD?

Reply:
yes, we do : [crd](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-worker/crds/networking.kubeslice.io_slicegateways.yaml#:~:text=sliceGatewayNodePort%3A)